WebRoot/js/zimbraAdmin/accounts/view/ZaNewAccountXWizard.js

```diff
@@ -1237,13 +1244,7 @@ ZaNewAccountXWizard.myXFormModifier = function(xFormObject, entry) {
                     });
 
         };
-        if(ZAWizTopGrouper_XFormItem.isGroupVisible(entry,[ZaAccount.A_zimbraFeatureMailPriorityEnabled,ZaAccount.A_zimbraFeatureFlaggingEnabled,
-            ZaAccount.A_zimbraImapEnabled,ZaAccount.A_zimbraPop3Enabled,
-            ZaAccount.A_zimbraFeatureWebClientEnabled,
-            ZaAccount.A_zimbraFeatureImapDataSourceEnabled,
-            ZaAccount.A_zimbraFeaturePop3DataSourceEnabled,ZaAccount.A_zimbraFeatureConversationsEnabled,ZaAccount.A_zimbraFeatureFiltersEnabled,
-            ZaAccount.A_zimbraFeatureOutOfOfficeReplyEnabled,ZaAccount.A_zimbraFeatureNewMailNotificationEnabled,
-            ZaAccount.A_zimbraFeatureMailSendLaterEnabled,ZaAccount.A_zimbraFeatureIdentitiesEnabled,ZaAccount.A_zimbraFeatureReadReceiptsEnabled],[])) {
+        if(ZAWizTopGrouper_XFormItem.isGroupVisible(entry,ZaNewAccountXWizard.MAIL_FEATURE_ATTRS,[])) {
```

Comment:
I understand that `ZaNewAccountXWizard.MAIL_FEATURE_ATTRS` needs to be declared so that other attribute(s) can be added from an admin zimlet.
Please update the description of the PR; the reason why the change is necessary. The current description `added mail recall attributes in the group visibility of Mail feature table of AccountXWizard.` is not exact.

Reply:
done